src/table.js

```diff
@@ -527,7 +524,7 @@ Table.prototype.createReadStream = function(options) {
       rowStream.unpipe(userStream);
       if (
         numRequestsMade <= maxRetries &&
-        HTTP_RETRYABLE_STATUS_CODES.has(error.code)
+        RETRYABLE_STATUS_CODES.has(error.code)
```

Comment:
We're no longer using `grpc-common` so GRPC error codes are no longer converted to HTTP error codes